student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly note that as this class enherits from Persone class its constructor should extend all parent's @instance variables comment in the children initialize parameter. :point_down: 

```rb
class Student < Person
   def initialize(age, classroom, name = "Unknown", parent_permission = true)  
        super(age, name, parent_permission)
          @classroom = classroom
    end
end
```